bin/remove-replacements.sh

```diff
@@ -0,0 +1,10 @@
+#! env bash
```

Comment:
Master script of commands that did the replacements. Once merged, can be deleted.